src/background.js

```diff
@@ -31,9 +31,11 @@ chrome.tabs.onAttached.addListener((tabId, data) => {
   // so we can do the same to the next one to open.
   chrome.tabs.get(tabId, tab => {
     if (tab.url.startsWith(URLS.ownOrigin + 'edit.html')) {
-      chrome.windows.get(tab.windowId, {populate: true}, win => {
+      chrome.windows.get(tab.windowId, {
+        populate: true
```

Comment:
This robotic expansion of literals doesn't make the code more readable. On the contrary, it's less readable in such cases as this one where only one parameter is used.